src/PatientTestsApi/Util/Utils.ts

```diff
@@ -0,0 +1,158 @@
+import moment from "moment";
```

Comment:
Can we trim this file down to just the functions that are in use, and if they're only being used in one place, move them back to that location? I think that will kill this file completely. I'm not a fan of a "random bag of things" file like this.